provider/etc/labels.conf

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2017 Google, Inc.
```

Comment:
Please squash your commits so that it starts out with copyright to Cloudera from the start.